Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs

```diff
@@ -1,29 +1,29 @@
 ﻿using System;
 using System.Linq;
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Promact.Trappist.DomainModel.Models.Question;
 using Microsoft.EntityFrameworkCore;
 using Promact.Trappist.Web.Models;
-using Promact.Trappist.DomainModel.Models.Question;
+using Promact.Trappist.DomainModel.Models.Category;
 using Promact.Trappist.DomainModel.Models;
 
-namespace Promact.Trappist.Web.Data
+namespace Promact.Trappist.Web.DomainModel.DbContext
```

Comment:
why again namespace got changed? how many times have I to comment about namespace?